clients/shared/table_config.go

```diff
@@ -102,25 +82,28 @@ func (g GetTableCfgArgs) GetTableConfig() (*types.DwhTableConfig, error) {
 			row[columnNameList[idx]] = strings.ToLower(fmt.Sprint(*interfaceVal))
 		}
 
-		kindDetails, err := g.Dwh.Dialect().KindForDataType(row[g.ColumnTypeLabel], row[constants.StrPrecisionCol])
+		kindDetails, err := g.Dwh.Dialect().KindForDataType(row[g.ColumnNameForDataType], row[constants.StrPrecisionCol])
 		if err != nil {
 			return nil, fmt.Errorf("failed to get kind details: %w", err)
 		}
+
 		if kindDetails.Kind == typing.Invalid.Kind {
-			return nil, fmt.Errorf("failed to get kind details: unable to map type: %q to dwh type", row[g.ColumnTypeLabel])
+			return nil, fmt.Errorf("failed to get kind details: unable to map type: %q to dwh type", row[g.ColumnNameForDataType])
 		}
 
-		col := columns.NewColumn(row[g.ColumnNameLabel], kindDetails)
-		comment, isOk := row[g.ColumnDescLabel]
-		if isOk && g.ShouldParseComment(comment) {
+		col := columns.NewColumn(row[g.ColumnNameForName], kindDetails)
+		if _, isOk := row[g.ColumnNameForDefaultValue]; isOk && isCommentSet(row[g.ColumnNameForDefaultValue]) {
+			col.SetBackfilled(true)
+		}
+
+		if comment, isOk := row[g.ColumnNameForComment]; isOk && isCommentSet(comment) {
```

Comment:
Should this be in an `else` block so that it doesn't run if col.SetBackfilled(true) has been set?